### ISSUE TITLE
Status of RSS-14 and RSS-Expanded readers

### ISSUE THREAD

**Issue by aryahz** [zxing-js/library#80]
What's the status of the RSS-14 and RSS-Expanded readers - are the readers going to be available soon?

**Comment by odahcam** [comment]
We have nothing scheduled for them yet. 😢 

I'm currently working on the library quality itself and accepting and giving support to ports from the original repo, so if someone make a PR with RSS-14 and RSS-Expanded readers certainty they gonna be available ASAP.

That's all for now, thanks for reaching out. Feel free to ask more questions.

**Comment by aryahz** [comment]
Thanks for the update. We really like what you've done so far with the library and are considering porting the RSS-14 and RSS-Expanded readers ourselves, as we need the functionality fairly soon. If we decide to go ahead I will make a PR.

**Comment by odahcam** [comment]
That's great! If you do so, please join our Gitter channel for better support. 🙂

**Comment by odahcam** [comment]
That's great! If you do so, please join our Gitter channel for better support. 🙂

**Comment by werthdavid** [comment]
@aryahz 
any decisions made yet?

**Comment by aryahz** [comment]
Yes, we've converted the RSS-14 reader classes and they're working in our test environment. See https://github.com/zxing-js/library/pull/114 for details.

**Comment by ilinkMat** [comment]
I tried Porting the RSS Expanded readers but I'm running in the some issues when importing the library in to a angular project. Could any one help me out here ? Please 

The command used to build  the ported  files is 
`yarn clean && yarn tsc && yarn webpack --mode production` 

Once the build process was successful , I copied the files from the folders 'esm5' & 'umd' added to the  node-modules of the angular project and I see the below errors on the console log 


![image](https://user-images.githubusercontent.com/35230376/49788831-63e58f80-fd50-11e8-836d-2846f79b6290.png)

**Comment by ilinkMat** [comment]
Thank you for you support. I did try the installation method which you suggested  as well . I wasn't able to proceed further. 

I suspect that the code which I have written to Port RSS Expanded Reader has circular dependencies in it which is why I see the errors. 

https://res.cloudinary.com/dzu1mlynh/image/upload/v1544602104/Screen_Shot_2018-12-12_at_1.24.18_PM.png

https://res.cloudinary.com/dzu1mlynh/image/upload/v1544602085/Zxing-1.png

**Comment by odahcam** [comment]
You seem right, you can open a [WIP] PR here so we can analyse the code together if you want, also our build system will automatically run on Travis every push you make.

**Comment by ilinkMat** [comment]
@odahcam I have created a pull request as requested . #120.
